automation/scripts/clean_pipelines/README.md

```diff
@@ -0,0 +1,90 @@
+# Clean Pipelines Script 
+
+This script is designed to delete old pipeline runs associated with a specific MLRun project,
+with flexible date filtering.
+You can use this as an MLRun function to automate pipeline cleanup tasks.
+
+## Usage
+
+### Step 1: Set up the project
+
+First, create or get an existing MLRun project:
+
+```python
+import mlrun
+
+project_name = "your_project_name"
+project = mlrun.get_or_create_project(project_name, context="./")
+```
+
+### Step 2: Set up the function and parameters
+
+Add the `clean_pipelines.py` script to your project at the location of your choice. Set up the function as follows:
+
+```python
+code_path = "/path/to/script/clean_pipelines.py"
+function_name = "function-name"
+
+func = project.set_function(
+    name=function_name,
+    func=code_path,
+    kind="job",
+    image="mlrun/mlrun",
+    handler="delete_project_old_pipelines",
+)
+```
+
+The `delete_project_old_pipelines` function deletes old pipeline runs associated with a specified project.
+
+#### Function parameters:
+- **context**: The context object to log results (automatically passed by MLRun).
+- **project_name** (str): The name of the project for which to delete old pipelines.
+- **end_date** (str): The cutoff date for deleting pipeline runs. All runs created on or before this date will be considered for deletion (format: YYYY-MM-DDTHH:MM:SSZ).
```

Comment:
Can `(format: YYYY-MM-DDTHH:MM:SSZ)` be translated to something readable? is it isoformat?